cypress/support/component.js

```diff
@@ -0,0 +1,27 @@
+// ***********************************************************
```

Comment:
Remove these files and we're good to go :upside_down_face:

Reply:
Hopefully I removed the correct files.